### ISSUE TITLE
Can't find anything in searchbox

### ISSUE THREAD

**Issue by abigger** [swissbib/vufind#468]
Hi Lionel,
I can't bring the searchbox you sent the link for in your mail to offer any hits. Is anything indexed at all?
Andreas

**Comment by guenterh** [comment]
@abigger @liowalter @edelm 
Hi Andreas
could it be the link is targeting to www.swissbib.ch?
Our switch to production is upcoming Monday so actually our productive service doesn't contain any National License data. I'm going to kick off the indexing on Sunday.
Other possibility:
if your link is targeting to test.swissbib.ch:
It might be that yesterday evening Matthias deployed a test installation for Basel / Bern on the host which was used for test.swissbib.ch some days ago. Therefor the new deployment is using another index which doesn't contain the NationalLicenses.

Hopefully things should be running as expected upcoming Monday as a productive service with the domain www.swissbib.ch

Best wishes from Basel and have a nice weekend!

Günter

**Comment by abigger** [comment]
@guenterh @liowalter  @edelm 
Oh, well, obviously it was to early in the morning for me (though I am an early bird normally ...)
However, the link Lionel provided to the consortium for testing was https://test.swissbib.ch/Search/Results?type=AllFields&filter%5B%5D=union%3A%22NATIONALLICENCE%22&daterange%5B%5D=publishDate&publishDatefrom=&publishDateto=2015 - i.e. test.swissbib.ch. And yes, it looked pretty much like swissbib Basel Bern to me.
The problem: We got the mail about this test only yesterday. So there was no way to test it earlier. But if everything is already tested now, there is no need to roll back for a further test.
With kind regards from a bright white morning on the Irchel hill
Andreas

**Comment by guenterh** [comment]
@abigger @liowalter @edelm @sonjakupfi 
Dear Andreas
I just deployed the NL branch again under the domain test.swissbib.ch because I heard from my colleague Sonja that Matthias used another host yesterday and because I got a white screen first hand I re-deployed it. But maybe it's better to wait with visiting the domain until next Monday where things should be in productive mode for all swiss private users. Then we welcome all feedback. 

We are in the phase where we are switching all our hosts from RedHat to Ubuntu. With more than 50 entities it might happen that sometimes things are a little bit in disorder - sorry.

Greetings from Basel under a blue sky!
Günter
